### PR TITLE
[#70] Remove language choice in menu

### LINKED ISSUE #68: "Set calendar" should be changed to just "Calendar"
From the menu view, the text should be changed from  "Set calendar"  to just "Calendar"

### LINKED ISSUE #70: CORRECTION: US6A - Language
- [ ] Remove language selection in app, simply check the phone's settings to determine what language the application should be in

### PR DESCRIPTION
**Pull request by eufekt**
- Remove language choice form menu
- Remove the redux stateListener and dispatcher for updating the language across components. This was the only example on how to use Redux.  Refer to the branch US6A - language before it gets deleted you can also consult with me (Nichita Hariton) on how to implement it or just search the web.
- Left the redux associated packages as well as the store because we will need it in the future.

- [#68] is in here as well. I changed Set Calendar to Calendar

### CODE REVIEW

File: store/index.js [hunk 1]
```diff
@@ -5,13 +5,16 @@ const initialState = {
...
-  if (action.type === 'CHANGE_LANGUAGE') {
-    return {
-      language: action.payload.language
-    };
-  }
+  // set action type here example:
+  // if (action.type === 'CHANGE_LANGUAGE') {
+  //   return {
+  //     language: action.payload.language
+  //   };
```
Comment (trevornag) [inline store/index.js]: should we remove the comments or do you think it's good to keep them?

**Review by h-karim — APPROVED** [review]
Languages menu seems to be effectively removed. #68 seems to be solved as well. 

Code removal seems to make sense. Approved

File: store/index.js (hunk 1 above)
Comment (eufekt) [reply store/index.js]: I removed it. Whoever will use redux will understand what are actions and TYPES.

**Review by trevornag — APPROVED** [review]

**Review by mohhef — APPROVED** [review]
Languages removed from side menu with no new introduction of bugs.

**Pull request merged**